pages/en/developers/apis/4-realtime-api/vehicle-positions/index.md

```diff
@@ -14,6 +14,19 @@ Most of the vehicles in the HSL area should publish their status, including thei
 The devices of the end users, e.g. smartphones, may subscribe to receive the relevant messages based on their context, e.g. filtered on the mode of transport, the route ID, the geographical region etc.
 The subscription scope is specified by the MQTT topic structure of the API.
 
+## Quickstart
+
+Try this example to get an quick idea of what kind of data is available from the API.
+
+1. Install [MQTT.js](https://github.com/mqttjs/MQTT.js) command line tools:  
+```
+npm install -g mqtt
+```
+2. Use MQTT.js to subscribe to HFP messages:  
+```
+mqtt subscribe -h mqtt.hsl.fi -v -t "/hfp/v1/journey/#"
```

Comment:
Actually, we should default to TLS. Maybe we'll deprecate unencrypted ports in the future.
```suggestion
mqtt subscribe -h mqtt.hsl.fi -p 443 --protocol mqtts -v -t "/hfp/v1/journey/#"
```